docs/en/writing-running-appium/other/troubleshooting.md

```diff
@@ -36,6 +36,8 @@ own items in the SDK manager. Make sure you install the build-tools and platform
   reset the connection to the Android device.
 * If you're running the Android emulator using Fastboot, try doing a cold boot (Android Studio > Tools > AVD Manager > Actions > Cold Boot Now)
 * Make sure you set ANDROID_HOME pointing to the Android SDK directory
+* After calling `driver.setPowerAC(PowerACState.OFF);` you must recreate your session.
+* When using `-wipe-data` in `AVD_ARGS` the `NO_RESET` capability will be ignored if set to true.
```

Comment:
usually capabilities are camelCase; are these constants from a particular client?

Reply:
Cool, in this doc we should probably not use constants from just a particular client since they will not be valid for people coming here using other clients. But looks like @dpgraham merged it already so we'll live with it :-) Thanks @ndinoia

Reply:
@jlipps I just put in another PR to refine this: https://github.com/appium/appium/compare/master...dpgraham-troubleshooting-tweaks?quick_pull=1